flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/factories/TestValuesManagedTableFactory.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.planner.factories;
+
+import org.apache.flink.table.factories.ManagedTableFactory;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/** Test implementation of {@link ManagedTableFactory}. */
+public class TestValuesManagedTableFactory extends TestValuesTableFactory
```

Comment:
Why needs implement `TestValuesTableFactory`?
Can we just use `TestManagedTableFactory` in common?

Reply:
@JingsongLi 
Sure, and shall we move all existing tests about `ManagedTableFactory`  from  `flink-table-common`  to `flink-table-planner`? For the three reasons:

- Avoid clash.
- Reuse code from TestValuesTableFactory.
- Just as you mentioned above, use TestManagedTableFactory in common.